tests/test_filter_handler.py

```diff
@@ -18,19 +18,176 @@ def emit(self, record):
 
 
 def test_filter():
-    filter_regex = r'/health|/metrics'
+    filter_regex = r"/health|/metrics"
 
-    handler = CollectingNamedCeeLogger(_DUMMY_HOST, _DUMMY_PROTOCOL, 'myname')
+    handler = CollectingNamedCeeLogger(_DUMMY_HOST, _DUMMY_PROTOCOL, "myname")
     handler.addFilter(RegexFilter(filter_regex))
 
-    handler.handle(makeLogRecord({'msg': 'POST /important/endpoint 200 OK'}))
-    assert len(handler.emitted_records) == 1 # record emitted
+    handler.handle(makeLogRecord({"msg": "POST /important/endpoint 200 OK"}))
+    assert len(handler.emitted_records) == 1  # record emitted
+
+    handler.handle(makeLogRecord({"msg": "GET /health 200 OK"}))
+    assert len(handler.emitted_records) == 1  # no new record emitted
+
+    handler.handle(makeLogRecord({"msg": "GET /metrics 404"}))
+    assert len(handler.emitted_records) == 1  # no new record emitted
+
+    handler.handle(
+        makeLogRecord(
+            {"msg": "Eating vegetables is healthy and improves blood metrics"}
+        )
+    )
+    assert len(handler.emitted_records) == 2  # new record emitted
+
+
+def test_redacting_filter():
```

Comment:
Please add one more test that tests the fast no-op case (where we bail early). Assumption would be that we don't modify the record and thus args are still there.

Reply:
I forgot that case :)